From b10341f5c13d71271e195101e46213026f628048 Mon Sep 17 00:00:00 2001
From: Dirk Mueller <dmueller@suse.com>
Date: Mon, 12 Jan 2015 15:52:34 +0100
Subject: [PATCH 15/33] Add 'licence' to the list of license files as well

This is a misspelling, but seems to be common enough still
---
 templates/gem_packages.spec.erb | 1 +
 templates/opensuse.spec.erb     | 1 +
 2 files changed, 2 insertions(+)

diff --git a/templates/gem_packages.spec.erb b/templates/gem_packages.spec.erb
index d3a43fd..b772d02 100644
--- a/templates/gem_packages.spec.erb
+++ b/templates/gem_packages.spec.erb
@@ -124,6 +124,7 @@ Usually in RDoc and RI formats.
         copying
         history
         legal
+        licence
         license
         license-mit
         mit-license
diff --git a/templates/opensuse.spec.erb b/templates/opensuse.spec.erb
index 88e7356..8d14e38 100644
--- a/templates/opensuse.spec.erb
+++ b/templates/opensuse.spec.erb
@@ -114,6 +114,7 @@ PreReq:         update-alternatives
         copying
         history
         legal
+        licence
         license
         license-mit
         mit-license
-- 
2.26.2

